regulations/static/regulations/css/scss/_base-variables.scss

```diff
@@ -57,6 +57,7 @@ $panel_text_color: $black; // Text color for table of contents
 $text_highlight_color: $green_lightest; // Text highlight used for diffs
 $success_color: $green_lightest; // Used as background for success messages
 $notice_color: $blue_lightest; // Used as background for neutral notice messages
+$alert_color: $orange_lightest; // Used as background for caution messages
```

Comment:
Would like this to be `#fff0dd` similar to the CFPB lightest orange, please.

Reply:
I think it's fine to change it to the originating `$orange_lightest;` declaration.

Reply:
What do you think about using the "gold" color scheme from the [USDWS](https://standards.usa.gov/components/colors/) (eRegs' base _should_ be following those color schemes, but it may have moved out from under us)? We'd be at `#fff1d2` then.

Reply:
Closing the loop, @micahtaylor said this was a reasonable solution.